<?xml version="1.0"?>
<launch>
  <include file="$(find kamu_robotu_launch)/launch/oko_description.launch"/>
  <node name="lidar_bridge" pkg="kamu_robotu_comm" type="lidar_bridge" respawn="false" output="screen"></node>
  <node name="odometry_bridge" pkg="kamu_robotu_comm" type="odometry_bridge" respawn="false" output="screen"></node>

</launch>
